lightning-invoice/src/utils.rs

```diff
@@ -0,0 +1,158 @@
+/// Convenient utilities to create an invoice.
```

Comment:
In order for this to be a doc comment it needs to be "//!" not "///"

Reply:
TIL